modules/client-lib/test/integration/index.test.ts

```diff
@@ -116,27 +117,29 @@ describe("client-lib-integration-tests", () => {
     await kafkaLogger.error("Logger message. Hello World! Lets error.");
     await kafkaLogger.fatal("Logger message. Hello World! Lets fatal.");
 
-    // Wait 1 second to receive the event
-    await new Promise(f => setTimeout(f, 1000));
-
-    expect(receivedMessages).toBe(6);
+    // Wait 10 second to receive the event
+    await new Promise(f => setTimeout(f, 5000));
+    
+    expect(receivedMessages).toBeGreaterThanOrEqual(6);
+    
   });
 
 
-  test("error object tests", async () => {
+  // test("error object tests", async () => {
```

Comment:
This is an important test, we should not remove it

Reply:
Ok. I removed that when I discussed some code issues with Jason. I will undo it.